bigbluebutton-html5/imports/ui/components/user-list/component.jsx

```diff
@@ -28,7 +28,7 @@ const propTypes = {
   changeRole: PropTypes.func.isRequired,
   roving: PropTypes.func.isRequired,
 };
-
+const SHOULD_SHOW_CUSTOM = Meteor.settings.public.app.branding.displayBrandingArea;
```

Comment:
Maybe a little more self explanatory constant, like `SHOW_BRANDING` or something like that.